cli/required.go

```diff
@@ -6,18 +6,10 @@ import (
 	"github.com/spf13/cobra"
 )
 
+// NoArgs checks that the command is not passed any parameters
 func NoArgs(cmd *cobra.Command, args []string) error {
 	if len(args) == 0 {
 		return nil
 	}
 	return fmt.Errorf("unexpected argument: %s\nSee '%s --help'", args[0], cmd.CommandPath())
 }
-
-func NoArgsCustom(errstr string) func(*cobra.Command, []string) error {
```

Comment:
I can do it, but if you don't mind, please add `// nolint: unused` above the func with your PR.